.env

```diff
@@ -1 +1 @@
-FLASK_ENV=development
\ No newline at end of file
+FLASK_ENV=production
```

Comment:
Not sure about all the side-effects of this change... but I don't think it's going to break anything.